data_steward/analytics/tools/visualizations/completeness/ingredient_usage_visualizations.py

```diff
@@ -101,7 +104,7 @@
 print(len(site_name_list))
 
 # +
-name_of_interest = 'aggregate_info'
+name_of_interest = 'nyc_cu'
 
```

Comment:
Is this supposed to be hardcoded to only work for `nyc_cu`?

Reply:
no it is something that we change to generate the graphs for each of the sites. looks like I happened to push when I just generated the graphs for nyc_cu. I can change this in the next PR but it does not affect functionality.